atomic_reactor/schemas/config.json

```diff
@@ -651,12 +651,20 @@
             "description": "Do not report any media types other than these",
             "allOf": [{"$ref": "#/definitions/media_types"}]
           },
-          "blacklist_srpms": {
-              "description": "Don't include specified srpms to source container image, specify SRPM name",
-              "type": "array",
-              "items": {
-                "type": "string"
-              }
+          "denylist_srpms": {
+              "description": "Don't include srpms specified in denylist json with key",
+              "type": "object",
+              "properties": {
+                  "denylist_url": {
+                    "description": "Url with denylist json file",
+                    "type": "string"
+                  },
+                  "denylist_key": {
+                    "description": "Key in denylist json file, with denylist sources as list, specify SRPM name",
```

Comment:
`, specify SRPM name` this seems extra

Reply:
Shall be this optional, i.e. no key defined the whole json doc will be used?

Reply:
there should be some key, I'd say